client/asset/btc/btc.go

```diff
@@ -4113,9 +4138,12 @@ func (btc *baseWallet) spendableUTXOs(confs uint32) ([]*compositeUTXO, map[outPo
 		// unlocked something or even restarted the wallet software.
 		pt := newOutPoint(utxo.txHash, utxo.vout)
 		if btc.fundingCoins[pt] != nil {
+			// This may happen Electrum, which returns even locked utxos in the
+			// listunspent RPC. Maybe we re-lockunspent if we hit this so
+			// electrumWallet's lockedOutpoints map is updated to prevent the
+			// listUnspent method from returning this again. Also, we should
+			// probably remove this outpoint from the slice and not return it!
 			btc.log.Warnf("Known order-funding coin %s returned by listunspent!", pt)
-			// TODO: Consider relocking the coin in the wallet.
```

Comment:
eh, removing the utxo from the return was what I was considering here.  Part of the unresolved coin locking issues for electrum...